docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -1596,8 +1596,35 @@ public class FooTest {
 <4> The test also injects `Charlie`, a dependency for which a synthetic `@Singleton` bean is registered automatically. The injected reference is an "unconfigured" Mockito mock.
 <5> We can leverage the Mockito API in a test method to configure the behavior.
 
-If you need the full control over the `QuarkusComponentTestExtension` configuration then you can use the `@RegisterExtension` annotation and configure the extension programatically.
-The test above could be rewritten like:
+`QuarkusComponentTestExtension` also resolves arguments for parameters of a test method and injects the matching beans.
```

Comment:
```suggestion
`QuarkusComponentTestExtension` also resolves parameters of test methods and injects matching beans.
```

Reply:
Hm, I tried to follow the wording from `org.junit.jupiter.api.extension.ParameterResolver` but I don't really care...

Reply:
Ah. Well I think their wording is only understandable by nerds (like me) who care deeply about the distinction between arguments and parameters :-)

If you really wanted to follow their wording, I would go with something like:

```
`QuarkusComponentTestExtension` also resolves arguments for parameters of a test method from by looking up a matching bean from the CDI container.
```

Or so.

Reply:
Well, your previous proposal was better ;-)